demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -177,11 +177,18 @@ def is_ask_condition_branches_handled(self, task: Dict) -> bool:
         # ADD all replyOptions to unhandled_reply_options (UPPER)
         unhandled_reply_options = set(map(str.upper, task.get('message', {}).get('replyOptions', [])))
 
+        # Rename the keys in dictionary to upper case
+        next_tasks_upper = {k.upper(): v for k, v in next_tasks.items()}
+
+        # Rename the dictionary keys from 'True Positive\False Positive' to 'YES\NO'
+        next_tasks_upper['YES'] = next_tasks_upper.pop('TRUE POSITIVE', next_tasks_upper.get('YES'))
+        next_tasks_upper['NO'] = next_tasks_upper.pop('FALSE POSITIVE', next_tasks_upper.get('NO'))
```

Comment:
What was the problem? 
Some of the tasks had 'TRUE/FALSE POSITIVE' while the others had 'YES/NO'?

Reply:
Exactly